### PR TITLE
LRSD-4693 Create Learn Education Home Page base structure

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/master-pages/liferay-learn-education/master-page.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ryanschuhler) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/master-pages/liferay-learn-education/master-page.json]: @luisacmn what is the reason we need a new master page? are we not able to do some of the requirements just in a new page?

Comment (luisacmn) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/master-pages/liferay-learn-education/master-page.json]: @ryanschuhler in Education pages, we don't use the capabilities and product tabs that is fixed on master page. 
[mock](https://www.figma.com/design/zXFxe3Skmm3jZjLQme5Pbm/Education-Home-Page-ISSD-4040?node-id=1-456)

Comment (luisacmn) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/master-pages/liferay-learn-education/master-page.json]: @ryanschuhler PR updated. Could you take a look ?

**Comment by ryanschuhler** [comment]
ci:forward

**Comment by ryanschuhler** [comment]
thanks! https://github.com/brianchandotcom/liferay-portal/pull/151043

**Pull request closed without merging**